src/js/configuration_panel/configuration-panel.js

```diff
@@ -85,14 +85,17 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                 closeCurrentWindow: "Close current browser window",
                 openNewWindow: "Open a new browser window",
                 goToPreviousWindow: "Switch to the previous browser window",
-                goToNextWindow: "Switch to the next browser window"
+                goToNextWindow: "Switch to the next browser window",
+                zoomIn: "Zoom-in on the current tab",
+                zoomOut: "Zoom-out on the current tab"
             },
             axes: {
                 null: "None",
                 scrollHorizontally: "Scroll horizontally",
                 scrollVertically: "Scroll vertically",
                 thumbstickHistoryNavigation: "History navigation",
-                thumbstickTabbing: "Focus on the previous/next element"
+                thumbstickTabbing: "Focus on the previous/next element",
+                thumbstickZoom: "Zoom in or out on the current tab"
```

Comment:
I'm wondering if this shouldn't be "tab/window", i.e. if there's only one tab.  I also think "active" is more helpful than "current".

Reply:
I think the "active webpage" would be better in that case. Should I replace the part of the description with it?

Reply:
sounds good.